client/lib/sites-list/list.js

```diff
@@ -101,6 +101,9 @@ SitesList.prototype.fetch = function() {
 SitesList.prototype.pauseFetching = function() {
 	this.ignoreUpdates = true;
 };
+SitesList.prototype.resumeFetching = function() {
+	this.ignoreUpdates = false;
```

Comment:
But we are not, since that approach didnt pan out, so instead of data-poller I'd try this